### DISCUSSION TITLE
Erro - Warning: dictionary has duplicated key - resulting file may have some problems

### DISCUSSION THREAD

**Discussion by SamirMMBr** [qpdf/qpdf#829] [Q&A]
Runed the command to create the encrypted file. Result is success.
Runed again and the result is a warning. **Why this message?** **How to resolve?**

qpdf.exe --encrypt User3Pw Adm4pW 256 -modify=none -- "C:\temp\01_047362_05.pdf" "C:\temp\01_047362_05_comsenha.pdf"
**WARNING**: C:\temp\01_047362_05.pdf (object 7 0, offset 33409): **dictionary has duplicated key /Im1; last occurrence overrides earlier ones**
qpdf: operation succeeded with warnings; **resulting file may have some problems**

**Comment by jberkenbilt** [comment]
I can't see any possible scenario where a file written by qpdf would generate that warning, so I think I may not be understanding what happens. Is `C:\temp\01_047362_05.pdf` the original file? Are you saying if you run the same command twice, you get a warning one time and not the other? Or are you saying the initial encrypt command works but rerunning qpdf on the output gives a warning (probably not possible -- qpdf wouldn't write a dictionary with two keys -- its internal model can't represent that).

If you just have a file like that, qpdf will give a warning, and you will just have to look at the file in a viewer to determine whether the output file is correct.
